web/src/tracking/activation.tsx

```diff
@@ -0,0 +1,153 @@
+import H from 'history'
+import { pick } from 'lodash'
+import { Observable } from 'rxjs'
+import { first, map } from 'rxjs/operators'
+import { ActivationStatus, ActivationStep } from '../../../shared/src/components/activation/Activation'
+import { dataAndErrors, dataOrThrowErrors, gql } from '../../../shared/src/graphql/graphql'
+import { queryGraphQL } from '../backend/graphql'
+
+/**
+ * Returns the URL to navigate to when the user clicks on the "Find references" activation step.
+ */
+const fetchReferencesLink: () => Observable<string | null> = () =>
+    queryGraphQL(gql`
+        query {
+            repositories(enabled: true, cloned: true, first: 100, indexed: true) {
+                nodes {
+                    url
+                    gitRefs {
+                        totalCount
+                    }
+                }
+            }
+        }
+    `).pipe(
+        map(dataAndErrors),
+        map(dataAndErrors => {
+            if (!dataAndErrors.data) {
+                return null
+            }
+            const data = dataAndErrors.data
+            if (!data.repositories.nodes) {
+                return null
+            }
+            const rURLs = data.repositories.nodes
```

Comment:
```suggestion
            const repositoryUrls = data.repositories.nodes
```

We like readable variable names in TypeScript :)